### DISCUSSION TITLE
提供翻譯檔案

### DISCUSSION THREAD

**Discussion by Liteya** [GpointChen/FFXIVChnTextPatch-GP#458] [Q&A]
請問如何提供翻譯過後的csv檔案給您做加入/參考之類的?

**Comment by GpointChen** [comment]
可以參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/4.-%E7%BF%BB%E8%AD%AF%E8%88%87%E5%95%8F%E9%A1%8C
或加我discord: gpointchen
